content/PIV/Guides/PIV_Walk-Through.adoc

```diff
@@ -17,64 +17,84 @@ The Yubico PIV tool is available link:https://developers.yubico.com/yubico-piv-t
 
 
 === Development Prerequisites
+Install:
+
+* OpenSSH
+* For OS X, iOS 10.13 or later.
+
 To implement the PIV security protocol in your app, ensure that your app:
 
 * Meets the link:https://csrc.nist.gov/projects/piv/piv-standards-and-supporting-documentation[Personal Identity Verification of Federal Employees and Contractors standards], link:https://csrc.nist.gov/publications/detail/fips/201/2/final[FIPS 201-2] standard, and the related mobile standard, link:https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.1800-12.pdf[Derived Personal Identity Verification (PIV) Credentials]
+
 * Works with a link:http://docs.oasis-open.org/pkcs11/pkcs11-base/v2.40/os/pkcs11-base-v2.40-os.html[PKCS#11 Cryptographic Token Interface Base Specification OASIS Standard].
 
 
 === Overview
-The steps to enable your app to use the PIV protocol vary depending upon the operating system. Any Certificate Authority (CA) considerations should be relegated to a separate exercise.
+The steps to enable your app to use the PIV protocol vary depending upon the operating system. Note that Certificate Authority (CA) considerations are not taken into account in this Walk-Through.
 
-#. Import or generate a key in the device slot
-#. Locate the module
-#. Export the public key
-#. Authenticate to the target system
-#. (Optional) Set it up to work with ssh-agent.
+1. Import or generate a key in the device slot
 
+2. Locate the module
 
-=== Configuration Prerequisites
-To manage the PIV security protocol on your PIV-compliant app, install the Yubico PIV tool and the Yubico PKCS#11 module, ykcs11, which is part of the PIV tool package, on the administrative system.
+3. Export the public key
 
+4. Authenticate to the target system
 
-=== Configure for SSH on PKCS#11
-Public key authentication with link:https://developers.yubico.com/PIV/Guides/SSH_with_PIV_and_PKCS11.html[OpenSSH through PKCS#11]. Examples for OS X and Linux systems.
+5. (Optional) Set it up to work with ssh-agent.
 
 
-==== Prerequisites
-In addition to those listed above, install:
+=== Configuration Prerequisites
+To manage the PIV security protocol on your PIV-compliant app, on the administrative system, install the Yubico PIV tool and the Yubico PKCS#11 module, ykcs11, which is part of the PIV tool package.
 
-* OpenSSH
-* For OS X, iOS 10.13 or later
+For SSH on PKCS#11, configure public key authentication with link:https://developers.yubico.com/PIV/Guides/SSH_with_PIV_and_PKCS11.html[OpenSSH through PKCS#11], which provides examples for OS X and Linux systems.
 
 
 === Generate or Import an SSH Private Key with a YubiKey
 
 *Step 1*: Import or generate a key in any slot. If an external key has been imported and a certificate exists, skip ahead to Step 2.
 
-To import the key (PEM format), run: ``$ yubico-piv-tool -s 9a -a import-key -i key.pem``
+To import the key (PEM format), run:
+
+....
+yubico-piv-tool -s 9a -a import-key -i key.pem
+....
+
+To generate the key, run:
```

Comment:
Same thing regarding the slot here

Reply:
I've added it to all the places where slot 9A is mentioned. However, I implied that it is evident that you load the certificate into the same slot as the key. If not, we should explain why you might put it in a different place.

Reply:
No, you are correct. The certificate should be loaded in the same slot otherwise it won't be usable. Good addition